tests/test_process_executor_fork.py

```diff
@@ -8,6 +8,7 @@
 if sys.version_info[:2] > (3, 3) and sys.platform != "win32":
     class ProcessPoolForkMixin(ExecutorMixin):
         executor_type = process_executor.ProcessPoolExecutor
+        start_method = "fork"
         context = mp.get_context('fork')
```

Comment:
Where is this new variable used? Why not use `context.get_start_method()` instead to avoid redundancy and potential discrepancy?